src/draw/DefaultDrawer.js

```diff
@@ -205,47 +205,54 @@ class DefaultDrawer {
   __dropToContainer(components) {
     let invalidDropzones;
     let currentComponent;
-    this.interact('.isContainer').unset();
-    this.interact('.isContainer').dropzone({
-      accept: '.component',
-      overlap: 'pointer',
-      ondropactivate: (event) => {
-        currentComponent = this.d3.select(`#${event.relatedTarget.id}`);
-        invalidDropzones = [];
-        if (currentComponent.datum().definition.isContainer) {
-          this.__getChildrenContainer(currentComponent.datum().children, invalidDropzones);
-        }
-      },
-      ondrop: (event) => {
-        if (!invalidDropzones.includes(event.target.id)) {
-          const dropzone = this.d3.select(`#${event.target.id}`).datum();
+    components.forEach((component) => {
```

Comment:
according to sonar, you have to rewrite this to remove complexity

Reply:
Done